src/fe-common/core/fe-common-core.c

```diff
@@ -462,25 +462,73 @@ void fe_common_core_finish_init(void)
 gboolean strarray_find_dest(char **array, const TEXT_DEST_REC *dest)
 {
 	g_return_val_if_fail(array != NULL, FALSE);
+	WI_ITEM_REC *item = window_item_find_window(dest->window, dest->server, dest->target);
+	int channel_type = module_get_uniq_id_str("WINDOW ITEM TYPE", "CHANNEL");
+	int query_type = module_get_uniq_id_str("WINDOW ITEM TYPE", "QUERY");
+	
+	char **tmp;
+	for (tmp = array; *tmp != NULL; tmp++) {
+		char *tagtarget;
```

Comment:
Set this to NULL, otherwise it's uninitialized access down there

Reply:
Done